docs/caveat-unique-values.doctest.rst.inc

```diff
@@ -0,0 +1,121 @@
+.. _caveat-unique-values:
+
+Values Must Be Unique
+---------------------
+
+Consider the following::
+
+    >>> from bidict import bidict
+    >>> b = bidict({0: 0, 1: 1})
+    >>> b[0] = 1  # doctest: +SKIP
+
+What should happen next?
+
+If the bidict allowed this to succeed,
+because of the uniqueness-of-values constraint,
+it would silently clobber both of the existing mappings,
```

Comment:
I don't think it would if users aren't familiar with the existing behaviour. The naive behaviour would result in

    bidict({0: 1, 1: 1})

...but this is obviously not invertible. Suggesting a clobbering here makes the reader think about a *new* set of semantics to what we're actually trying to describe. I think we should just talk about the semantics bidict does have.